### PR TITLE
Lint Thai for real ellipsis

### LINKED ISSUE #269: Lint for standardized Ellipses
This urgently applies to LBF, but will be used in PP as soon as LBF is finished. I believe that @hsteck1844 decided against using the unicode ellipses character in MB and favored 3 dots. There are outlier cases where the ellipsis follows a '... closing quotation mark etc. I think I have been spacing ` ... ` around three periods, but again, we need to standardize.

### PR DESCRIPTION
**Pull request by mattleff**
Fixes #269.

### CODE REVIEW

File: lo/GC/001_machineraw/GC21_lo.md
Comment (bountonw) [inline lo/GC/001_machineraw/GC21_lo.md:1]: @mattleff are we sure we have the code right? It changed `....` to `….` I didn't check further. But I know there are some ."... and !... etc. in the mix.

**Review by bountonw — COMMENTED** [review]
@mattleff probably better to comment here instead of in the file. 

I'm currently working on translating GC29. In this file I see `:....` `;....`

On second thought. Ignore the Lao files. They will need to be handled differently since Lao uses the full range of English punctuation. For Thai we need to strip out everything but the ellipsis and I think we need to standardize to the three dot ellipsis character (since Thai doesn't have sentence ending punctuation.)

**Comment by mattleff** [comment]
@bountonw I've changed this PR to only lint Thai for real ellipsis.

**Comment by mattleff** [comment]
@bountonw Bump for review. This PR only changes Thai files now. I need this on main or I'll have to revert the lint commit in the tooling repo for now.

**Comment by bountonw** [comment]
There are inline comments from my previous review. I don't see the changes.

**Comment by mattleff** [comment]
@bountonw I removed all the changes to the Lao files which is where we had the ellipsis stacked with other punctuation (such as `….` or `…!`). I also changed the linting to only apply to Thai files (for now).

**Comment by bountonw** [comment]
@mattleff Did you see my inline comments?

**Comment by mattleff** [comment]
@bountonw I think so, regarding changing `....` to `….`, right? But Thai doesn't have any punctuation use with ellipsis and I'm only dealing with Thai on this PR.

File: th/PP/PP5/02_edit/PP62_th.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ chapter:
 
 ## {PP 637.2}
 
-ในขณะที่ดาวิดใช้ชีวิตสมถะเรียบง่ายตามประสาผู้เลี้ยงแกะอยู่นั้น พระเจ้าตรัสกับผู้เผยพระวจนะซามูเอลเกี่ยวกับเขา “องค์พระผู้เป็นเจ้าตรัสกับซามูเอลว่า ‘เจ้าจะคร่ำครวญเรื่องที่เราถอดซาอูลออกจากตำแหน่งกษัตริย์อิสราเอลไปอีกนานเท่าใด บัดนี้จงเติมน้ำมันให้เต็มเขาสัตว์ และออกเดินทาง เราจะส่งเจ้าไปพบเจสซีแห่งเบธเลเฮม เราได้เลือกบุตรชายคนหนึ่งของเขาให้เป็นกษัตริย์’... ‘จงเอาวัวตัวเมียไปด้วยตัวหนึ่ง และพูดว่า “ข้าพเจ้ามาเพื่อถวายเครื่องบูชาแด่องค์พระผู้เป็นเจ้า” จงเชิญเจสซีไปร่วมการถวายเครื่องบูชา แล้วเราจะบอกให้เจ้ารู้ว่าจะต้องทำอะไรบ้าง เจ้าจงเจิมตั้งคนที่เราบอกไว้สำหรับเรา’ ซามูเอลก็ปฏิบัติตามพระดำรัสสั่งขององค์พระผู้เป็นเจ้า เมื่อเขามาถึงเบธเลเฮมบรรดาผู้อาวุโสของเมืองนั้นหวาดกลัวจนตัวสั่นและถามเขาว่า ‘ท่านมาอย่างสันติหรือ’ ซามูเอลตอบว่า ‘ถูกแล้ว เรามาอย่างสันติ’” (1 ซามูเอล 16:1–5 TNCV) ผู้อาวุโสเหล่านั้นรับคำเชิญให้ไปร่วมพิธีถวายบูชา แล้วซามูเอลก็เรียกให้เจสซีพร้อมกับบรรดาลูกชายของเขามา มีการสร้างแท่นบูชา ส่วนเครื่องบูชาก็ตระเตรียมไว้พร้อมแล้ว ทุกคนในครัวเรือนของเจสซีก็อยู่กันพร้อมหน้าพร้อมตายกเว้นดาวิดบุตรคนเล็กที่ถูกมอบหมายให้เฝ้าฝูงแกะ เพราะเป็นการไม่ปลอดภัยที่จะทิ้งฝูงแกะไว้โดยไม่มีคนดูแล {PP 637.2}
```
Comment (bountonw) [inline th/PP/PP5/02_edit/PP62_th.md:17]: Here is another example of a space after the ellipsis. This is different from the other issues.

**Review by bountonw — COMMENTED** [review]
@mattleff there are unresolved I line comments. Basically there are spaces before, after, or surrounding some ellipses. This needs to be standardized.

**Comment by mattleff** [comment]
@bountonw Ah, I see a bunch of comments now. Ok, I can work on this. Are we wanting to eliminate all spaces before and after ellipsis, or is it more complicated?

**Comment by bountonw** [comment]
I think we want to eliminate all of them. Is there an exception?

**Comment by mattleff** [comment]
@bountonw Check out ba69a32 and make sure this is looking right to you. I'll resolve merge conflicts if the PR looks good to go otherwise.

**Comment by bountonw** [comment]
@mattleff Looks good.

**Review by bountonw — APPROVED** [review]

**Pull request merged**